### DISCUSSION TITLE
Generated markdown files

### DISCUSSION THREAD

**Discussion by zmeers** [mkdocstrings/mkdocstrings#506] [Q&A] [answered]
Hi!

Great package, it's really helpful.
I'm wondering if it's possible to generate markdown files for docstrings instead of a compiled html site? I can't find anything in the instructions, either for `mkdocstrings` or `mkdocs`... But I figured I might be missing something!

Thanks

**Comment by pawamoy** [comment, marked as answer]
Hi, thanks for the kind words!

A similar question was asked here: https://github.com/mkdocstrings/mkdocstrings/discussions/346.
You might find interesting information there.

The short answer is: no, it's not possible. We couldn't have the features and visuals we have with just Markdown, so we use HTML directly.

**Reply by pawamoy** [reply]
There's now https://mkdocstrings.github.io/griffe2md/, only available to sponsors though.

**Comment by zmeers** [comment]
Thank you! It looks like `pydoc-markdown` does what I'm looking for in case anyone else is looking for something similar.
